File: spec/src/test/java/co/mscp/medicom/appointment/spec/CucumberRunner.java
package co.mscp.medicom.appointment.spec;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

@RunWith(Cucumber.class)
@CucumberOptions(
    plugin = {"pretty"},
    glue = "co.mscp.medicom.appointment.spec",
    features = "src/test/feature")
public class CucumberRunner {

}

File: spec/src/test/scala/co/mscp/medicom/appointment/spec/CucumberRunner.scala
package co.mscp.medicom.appointment.spec

import cucumber.api.CucumberOptions
import cucumber.api.junit.Cucumber
import org.junit.runner.RunWith

@RunWith(classOf[Cucumber])
@CucumberOptions(
  plugin = Array("pretty"),
  glue = Array("co.mscp.medicom.appointment.spec"),
  features = Array("src/test/feature"))
class CucumberRunner